### ISSUE TITLE
Error: Invalid value "copy" for "--dolby-vision-profile" - Am I missing something?

### ISSUE THREAD

**Issue by quamt** [rigaya/NVEnc#609]
When I try to use the option  `--dolby-vision-profile copy` in NVENC [7.59](https://github.com/rigaya/NVEnc/releases/tag/7.59)

GPU: NVIDIA GeForce RTX 3080 (8704 cores, 1725 MHz) [PCIe4x16]
Driver Version: 556.12
Operating System: Windows
NVENC: 7.59

I get the error message:

```
Error: Invalid value "copy" for "--dolby-vision-profile"
  Option value should be one of below...
    undef, unknown, auto, auto_res, bt709, smpte170m, bt470m, bt470bg,
    smpte240m, film, bt2020, st428, st431-2, st432-1, ebu3213-e
```

I'm also using:

```
--codec hevc
--output-depth 10
--colormatrix auto 
--colorprim auto 
--transfer auto
--chromaloc auto
--max-cll copy
--master-display copy
--atc-sei auto
--dhdr10-info copy
--repeat-headers
--dolby-vision-rpu copy
```

Am I missing something?

The input video as shown in MediaInfo:

```
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L5.1@High
HDR format                               : Dolby Vision, Version 1.0, dvhe.07.06, BL+EL+RPU, Blu-ray compatible / SMPTE ST 2086, HDR10 compatible
Codec ID                                 : V_MPEGH/ISO/HEVC
```

I

Please advise if I am overlooking anything or if there are additional steps I should take. Thank you for your assistance.

**Comment by quamt** [comment]
Im closing this now as it seems to been a driver issue.

Now 560.70 all ok.

**Comment by rigaya** [comment]
Might be using old NVEnc, as the wrong error message showing  list for colorprim was fixed in NVEnc 7.58.

**Comment by quamt** [comment]
Thank you. 
 I need to cleanup my machines :-)
